planning_launch/launch/scenario_planning/lane_driving/motion_planning/motion_planning.launch.py

```diff
@@ -83,43 +83,30 @@ def generate_launch_description():
         surround_obstacle_checker_param = yaml.safe_load(f)["/**"]["ros__parameters"]
     surround_obstacle_checker_component = ComposableNode(
         package="surround_obstacle_checker",
-        plugin="SurroundObstacleCheckerNode",
+        plugin="surround_obstacle_checker::SurroundObstacleCheckerNode",
```

Comment:
Please also update `tier4_planning_launch` in https://github.com/autowarefoundation/autoware.universe/pull/830. :bow:

Reply:
Oh, I missed it !! Thanks :+1: